patches/chromium/revert_add_ichromeaccessible.patch

```diff
@@ -0,0 +1,61 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@github.com>
+Date: Fri, 29 May 2020 13:47:46 -0400
+Subject: revert Add IChromeAccessible
+
+This was added in https://chromium-review.googlesource.com/c/chromium/src/+/2206224
+but it breaks WOA builds because
+third_party/win_build_output/midl/ui/accessibility/platform/arm64 does not exist.
+The link above says that the new interface is behind a feature flag which is
+disabled by default so it is safe to remove for now.
+
+diff --git a/ui/accessibility/BUILD.gn b/ui/accessibility/BUILD.gn
+index c1b07e18ae6ac6a5b58e3caf9ed268c2f02fb313..476a3c55375c8c052e1719e6757d2072aa354d12 100644
+--- a/ui/accessibility/BUILD.gn
++++ b/ui/accessibility/BUILD.gn
+@@ -16,10 +16,6 @@ if (is_android) {
+   import("//build/config/android/rules.gni")
+ }
+ 
+-if (is_win) {
+-  import("//build/toolchain/win/midl.gni")
+-}
+-
+ # Reset sources_assignment_filter for the BUILD.gn file to prevent
+ # regression during the migration of Chromium away from the feature.
+ # See docs/no_sources_assignment_filter.md for more information.
+@@ -74,14 +70,6 @@ jumbo_component("ax_base") {
+   ]
+ }
+ 
+-#if (is_win) {
```

Comment:
Isn't this a commented block ? why does it affect the build ?

Reply:
@deepak1556 I just backed it out since it was part of the original commit, but you are right it shouldn't affect the build. Also, I figured it might catch our attention on a patch conflict if it is enabled at some point.

Reply:
I am bit confused here, if this was originally commented out which is the template responsible for creating `third_party/win_build_output/midl/` , then what was the build failure that initiated the creation of this patch ?

Reply:
@deepak1556 you can see the build failure here: 
https://ci.appveyor.com/project/electron-bot/electron-ldhmv/builds/33200045.

The target  "//ui/accessibility/platform:ichromeaccessible", ends up trying to pull in stuff from src\third_party\win_build_output\midl\ui\accessibility\platform\arm64 but that dir doesn't exist

Reply:
Thanks for the context! can you raise an upstream issue for this, WoA is a supported architecture in upstream, they should be made aware of this failure.

Reply:
@deepak1556 looks like @richard-townsend-arm already did: https://bugs.chromium.org/p/chromium/issues/detail?id=1087866&q=OS%3DWindows%20arm64&can=2

Reply:
Awesome!

Reply:
I apologise folks, there’s a lot of stuff happening atm that has been drawing my attention from the WoA build. I’ll try and get it fixed ASAP

Reply:
@richard-townsend-arm no hurries, we just wanted to make sure upstream is aware of this so that an eventual fix can land there. Thanks for your work on this!

Reply:
Cool, so this CL should fix the issue: https://chromium-review.googlesource.com/c/chromium/src/+/2222349, hopefully it will land soon

Reply:
CL has now landed: https://chromium.googlesource.com/chromium/src/+/1b9e01844e8bf1aaafc4a52c0c62af7f56d9637b

Reply:
Thats the fastest turnaround 😄 Thanks!

Reply:
Thanks for the quick work on this @richard-townsend-arm!